common/models/express-usage-record.json

```diff
@@ -15,6 +15,33 @@
       "type": "date",
       "required": true
     },
+    "requestMethod": {
+      "type": "string"
+    },
+    "requestUrl": {
+      "type": "string"
+    },
+    "clientAddress": {
+      "type": "string"
+    },
+    "clientUsername": {
+      "type": "string"
+    },
+    "clientId": {
+      "type": "string"
+    },
+    "clientDetail": {
+      "type": "object"
+    },
+    "lbModelName": {
+      "type": "string"
+    },
+    "lbInstanceId": {
+      "type": "string"
+    },
+    "lbRemoteMethod": {
+      "type": "string"
+    },
```

Comment:
One more thing to consider: is it necessary to promote the nested "details" properties to the top level? Is it because LoopBack does not support nested properties for sorting? (That's something we should fix IMO.) Or is this an API design decision, to make it clear what record properties are always available?